pkg/virt-handler/container-disk/mount.go

```diff
@@ -26,18 +27,21 @@ import (
 //go:generate mockgen -source $GOFILE -package=$GOPACKAGE -destination=generated_mock_$GOFILE
 
 type mounter struct {
-	podIsolationDetector   isolation.PodIsolationDetector
-	mountStateDir          string
-	mountRecords           map[types.UID]*vmiMountTargetRecord
-	mountRecordsLock       sync.Mutex
-	suppressWarningTimeout time.Duration
-	pathGetter             containerdisk.SocketPathGetter
+	podIsolationDetector       isolation.PodIsolationDetector
+	mountStateDir              string
+	mountRecords               map[types.UID]*vmiMountTargetRecord
+	mountRecordsLock           sync.Mutex
+	suppressWarningTimeout     time.Duration
+	SocketPathGetter           containerdisk.SocketPathGetter
+	KernelBootSocketPathGetter containerdisk.KernelBootSocketPathGetter
```

Comment:
Do you need these attributes to be public? or can it be kernelBootSocketPathGetter and socketPathGetter?

Reply:
good catch, thanks!